helpers/UtilsController.h

```diff
@@ -0,0 +1,190 @@
+#pragma once
```

Comment:
We shouldn't use JSON-RPC for checking plugin status and activation. Pls use COM-RPC APIs similar to https://github.com/rdkcentral/rdkservices/blob/main/DisplaySettings/DisplaySettings.cpp#L215

Reply:
Updated  UtilsController.h to use com-rpc for activation.